content/creator/scenes/publishing/deploy-third-party.md

```diff
@@ -12,7 +12,7 @@ type: Document
 url: /creator/development-guide/deploy-third-party
 ---
 
-If you don't own parcels in Decentraland or are not ready to [deploy]({{< ref "/content/creator/scenes/publishing/publishing.md#the-test-server" >}}) your scene to Decentraland, you can upload your scene preview to run as an app in a free server.
+If you do not possess any parcels in Decentraland or a Decentraland [NAME](https://builder.decentraland.zone/names) to publish your scene to a [Worlds]({{< ref "/content/creator/worlds/about.md" >}}), or if you are not yet prepared to [deploy]({{< ref "/content/creator/scenes/publishing/publishing.md#the-test-server" >}}) your scene to Decentraland, there is an alternative option available. You can upload a preview of your scene to run as an application on a free server.
```

Comment:
I think Heroku is no longer free.. so I would change this "run as an application on a free server" to "run as an application on a Heroku server"